control.go

```diff
@@ -48,8 +49,10 @@ type Process struct {
 type Cgroup interface {
 	// New creates a new cgroup under the calling cgroup
 	New(string, *specs.LinuxResources) (Cgroup, error)
-	// Add adds a process to the cgroup
+	// Add adds a process to the cgroup (cgroup.procs)
 	Add(Process) error
+	// Add adds a process to the cgroup (tasks)
```

Comment:
The comment should be `AddTask adds a process..`; we don't have the linter running here, but that would be required for the comment to match the function name

Reply:
Thanks @estesp :+1: , code updated :smile: , please help to check again~